src/main/lrsql/admin/interceptors/openapi.clj

```diff
@@ -0,0 +1,11 @@
+(ns lrsql.admin.interceptors.openapi
+  (:require [io.pedestal.interceptor :refer [interceptor]]))
+
+(def openapi
+  "Returns an openapi spec in json form for lrsql"
+  (interceptor
+   {:name ::openapi
+    :enter
+    (fn openapi [_ctx]
+      {:status 200
+       :body (slurp "/resources/doc/openapi.json")})}))
```

Comment:
It looks like this will hit the filesystem on every request. That might be desirable in development when working on the schema itself but we probably want to def it into memory and just serve that.

Reply:
I debated about this---do we?  I had a vision of lrs sitting there in production, all nicely configured, but w/the file in memory *forever*.  Happy to memoize it, but...do we?

Reply:
@invaliduser Yes because it's looking in resources classpath for the file. We are not gonna do surgery on running Jars to fix doc. Additionally, changing the docs means changing the API, and that by definition cannot be done without a restart. On the flip side i don't care passionately about the performance of a non-operational endpoint. Overall I vote Memo.